packages/@sanity/color-input/src/ColorInput.js

```diff
@@ -84,7 +85,7 @@ export default class ColorInput extends PureComponent {
   render() {
     const {type, readOnly, value} = this.props
     return (
-      <Fieldset legend={type.title} description={type.description}>
+      <Fieldset legend={type.title} description={type.description} level={0}>
```

Comment:
I think we should rather forward `props.level` here instead of hard coding it to `0`

Reply:
Thanks, fixing! 👍